### PR TITLE
Update MyLinkedList.java

### PR DESCRIPTION
**Pull request by Sasha192**

### CODE REVIEW

File: src/main/java/core/basesyntax/MyLinkedList.java [hunk 1]
```diff
@@ -3,48 +3,172 @@
...
     public T get(int index) {
-        return null;
+        checkIndex(index);
+        return node(index).item;
     }
 
     @Override
     public void set(T value, int index) {
+        checkIndex(index);
+        Node ptr = node(index);
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/MyLinkedList.java]: @Sasha192 ptr?)

File: src/main/java/core/basesyntax/MyLinkedList.java [hunk 2]
```diff
@@ -3,48 +3,172 @@
...
+        ptr.item = value;
+    }
+
+    private void checkIndex(int index) {
+        if (index >= size || index < 0) {
+            throw new IndexOutOfBoundsException();
+        }
+    }
 
+    private Node<T> node(int index) {
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/MyLinkedList.java]: @Sasha192 please add more informative method name. All method names should starts from a verb

File: src/main/java/core/basesyntax/MyLinkedList.java (hunk 1 above)
Comment (boroda4436) [reply src/main/java/core/basesyntax/MyLinkedList.java]: @Sasha192 please verify that you have pushed new commit with fixes

Comment (Sasha192) [reply src/main/java/core/basesyntax/MyLinkedList.java]: > @Sasha192 please verify that you have pushed new commit with fixes

 i ' m sorry )

**Pull request closed without merging**